_pages/ato/levels.md

```diff
@@ -23,23 +25,23 @@ To categorize your system, go through each of the security objectives and determ
+The canonical or singular nature of a function being provided by the system must be taken into consideration in the categorization. The more singular and canonical the system under evaluation is, the higher the impact level.
 
 For example, if we re-post data from weather.gov, it is less impactful for us to lose availability than it is for weather.gov itself. Conversely, GSA is the only source of FedBizOpps data - therefore our availability is much more important for that data and function, and we should select a higher impact level for `availability`.
 
-### Conditions
+Storing [PII](../../security/pii/) always raises the level to at least `moderate` for the `confidentiality` and `integrity` objectives.
 
-* [Sensitive PII](../../security/pii/) always raises the level to at least `moderate` for the `confidentiality` and `integrity` metrics.
-* If there is any authorization or authentication being done, it is _likely_ at the `moderate` level for all metrics.
-* Just because we need `availability: high`, doesn't mean it needs `confidentiality: high` or `integrity: high`. These determinations are important for later tailoring of system controls.
+Just because we need `availability: high`, doesn't mean it needs `confidentiality: high` or `integrity: high`. These determinations are important for later tailoring of system controls.
 
 ### Selecting the overall impact level
 
-Once you have decided on the impact level (`low`, `moderate`, `high`) for each of three metrics (`confidentiality`, `integrity`, and `availability`), you must then determine the overall impact level of the system. A _low impact system_ is one in which all three of the security objectives are low. A _moderate impact system_ is one in which at least one of the objectives is moderate, and none are greater than moderate. A _high impact system_ is one in which at least one objective is high. Finally, the GSA TTS recognizes a category of system classified as _open data_, in which the integrity and availability objectives are classified as `low`, and there is zero confidentiality impact because the system contains only [Open Data](#open-data-atos). 
+Once you have decided on the impact level (`low`, `moderate`, `high`) for each of three objectives (`confidentiality`, `integrity`, and `availability`), you must then determine the overall impact level of the system. A _low impact system_ is one in which all three of the security objectives are `low`. A _moderate impact system_ is one in which at least one of the objectives is `moderate`, and none are `high`. A _high impact system_ is one in which at least one objective is `high`. Finally, the 18F recognizes a category of system classified as _open data_, in which the integrity and availability objectives are classified as `low`, and there is zero confidentiality impact because the system contains only [Open Data](#open-data-atos).
```

Comment:
Wouldn't `open data` now be a TTS thing, given that the Infrastructure team moved to that level?